### DISCUSSION TITLE
Add SECURITY.md

### DISCUSSION THREAD

**Discussion by sdushantha** [ghostty-org/ghostty#3713] [Feature Requests, Ideas]
The `SECURITY.md` file has become a standard for open source projects. This file can contain something very simple like an e-mail address with some basic steps to follow in order report a vulnerability. Ollama's `SECURITY.md` can be used as an inspiration: https://github.com/ollama/ollama/security

**Comment by ognevsd** [comment]
Here is also some info from OSSF scorecard: https://github.com/ossf/scorecard/blob/main/docs/checks.md#security-policy
